Documentation/project-docs/profiling-api-status.md

```diff
@@ -16,6 +18,10 @@ The implementation of this API was making some questionable assumptions about Wi
 
 See github issue [#13153](https://github.com/dotnet/coreclr/issues/13153) for more details.
 
+### ReJIT on ARM
+
+ReJIT optimizations are only available on x86/x64 for now. 
```

Comment:
Does this mean that the feature works fine, but it has worse performance than on x86/x64?

Reply:
Oh. I meant that ReJIT feature itself is not working on ARM. I'll change the word "optimizations" to "feature" to avoid ambiguity. Thank you!